# 10 Strategic Analysis

## Purpose

This notebook integrates the findings from the Germany and EU Tourism Competitiveness Analysis into a structured strategic assessment.

Rather than introducing new statistical tests or predictive models, the analysis synthesizes evidence from:

- Germany's historical tourism performance
- EU27 competitiveness benchmarking
- Source-market performance
- Average length of stay
- Travel economic indicators
- January–June 2026 performance
- Predictive modeling and the 2026 baseline scenario

The objective is to translate the analytical findings into practical tourism priorities while maintaining a clear distinction between observed evidence, interpretation, and strategic recommendation.

## Central Business Question

**How competitive is Germany's tourism sector compared with other EU destinations, what factors explain its current position, and where are the strongest opportunities for future tourism growth?**

## Strategic Framework

The analysis evaluates Germany across four dimensions:

### 1. Scale
How strong is Germany's position in attracting international visitors?

### 2. Tourism Depth
How effectively does visitor volume translate into overnight stays and longer stays?

### 3. Market Opportunity
Which international source markets offer the strongest combination of scale, growth, and stay-duration potential?

### 4. Momentum and Economic Context
Do the latest tourism and travel-economic indicators suggest strengthening, stagnation, or emerging risks?

Recommendations are evidence-based but should not be interpreted as causal claims. Where the analysis uses benchmark or scenario calculations, these are treated as decision-support tools rather than forecasts of guaranteed outcomes.

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

from pathlib import Path

pd.set_option(
    "display.max_columns",
    None
)

pd.set_option(
    "display.max_rows",
    100
)

project_root = Path.cwd().parent

processed_dir = (
    project_root
    / "data"
    / "processed"
)

figures_dir = (
    project_root
    / "figures"
)

In [2]:
eu_benchmark = pd.read_csv(
    processed_dir
    / "eu27_tourism_benchmark_2024.csv"
)

los_counterfactual = pd.read_csv(
    processed_dir
    / "germany_los_counterfactual_scenarios_2024.csv"
)

source_markets = pd.read_csv(
    processed_dir
    / "germany_major_source_market_strategy_2024.csv"
)

source_roles = pd.read_csv(
    processed_dir
    / "germany_source_market_strategic_roles_2024.csv"
)

travel_bop = pd.read_csv(
    processed_dir
    / "germany_travel_bop_summary_2021_2025.csv"
)

ytd_summary = pd.read_csv(
    processed_dir
    / "germany_2026_ytd_summary.csv"
)

ml_scenario = pd.read_csv(
    processed_dir
    / "germany_2026_baseline_scenario_summary.csv"
)

print(
    "EU benchmark:",
    eu_benchmark.shape
)

print(
    "LOS scenarios:",
    los_counterfactual.shape
)

print(
    "Major source markets:",
    source_markets.shape
)

print(
    "Strategic source roles:",
    source_roles.shape
)

print(
    "Travel BOP:",
    travel_bop.shape
)

print(
    "2026 YTD:",
    ytd_summary.shape
)

print(
    "2026 baseline scenario:",
    ml_scenario.shape
)

EU benchmark: (27, 11)
LOS scenarios: (6, 5)
Major source markets: (16, 16)
Strategic source roles: (7, 7)
Travel BOP: (5, 12)
2026 YTD: (2, 10)
2026 baseline scenario: (5, 2)


In [3]:
for name, df in {
    "EU Benchmark": eu_benchmark,
    "LOS Counterfactual": los_counterfactual,
    "Source Markets": source_markets,
    "Source Roles": source_roles,
    "Travel BOP": travel_bop,
    "2026 YTD": ytd_summary,
    "ML Scenario": ml_scenario
}.items():

    print(
        f"\n{name}"
    )

    print(
        df.columns.tolist()
    )

    display(
        df.head()
    )


EU Benchmark
['country_code', 'country', 'foreign_arrivals', 'arrivals_rank', 'foreign_nights', 'nights_rank', 'avg_length_of_stay', 'los_rank', 'scale_position', 'stay_position', 'competitive_position']


,country_code,country,foreign_arrivals,arrivals_rank,foreign_nights,nights_rank,avg_length_of_stay,los_rank,scale_position,stay_position,competitive_position
0,AT,Austria,28498633.0,5,94009113.0,5,3.298724,10,High Scale,Longer Stay,High Scale / Longer Stay
1,BE,Belgium,9638187.0,11,22279289.0,12,2.311564,20,High Scale,Shorter Stay,High Scale / Shorter Stay
2,BG,Bulgaria,2994009.0,19,15058045.0,18,5.029392,3,Low Scale,Longer Stay,Low Scale / Longer Stay
3,CY,Cyprus,3030161.0,18,16558854.0,15,5.464678,2,Low Scale,Longer Stay,Low Scale / Longer Stay
4,CZ,Czechia,10477528.0,10,25474111.0,10,2.431309,16,High Scale,Shorter Stay,High Scale / Shorter Stay



LOS Counterfactual
['benchmark', 'benchmark_los', 'illustrative_nights', 'additional_nights_vs_actual', 'additional_nights_pct']


,benchmark,benchmark_los,illustrative_nights,additional_nights_vs_actual,additional_nights_pct
0,Germany Actual,2.265983,8.479264e+07,0.000000e+00,0.000000
1,EU27 Median,2.490083,9.317841e+07,8.385770e+06,9.889738
2,France,2.577033,9.643208e+07,1.163944e+07,13.726946
3,Netherlands,2.894230,1.083015e+08,2.350887e+07,27.725134
4,Austria,3.298724,1.234376e+08,3.864494e+07,45.575818



Source Markets
['source_market_code', 'source_market', 'market_type', 'foreign_nights', 'foreign_arrivals', 'germany_foreign_nights_share_pct', 'published_market_share_pct', 'growth_2023_2024_pct', 'absolute_growth_nights', 'avg_length_of_stay', 'high_scale', 'high_growth', 'longer_stay', 'opportunity_score', 'strategic_role', 'stay_extension_priority']


,source_market_code,source_market,market_type,foreign_nights,foreign_arrivals,germany_foreign_nights_share_pct,published_market_share_pct,growth_2023_2024_pct,absolute_growth_nights,avg_length_of_stay,high_scale,high_growth,longer_stay,opportunity_score,strategic_role,stay_extension_priority
0,US,United States,individual_country,7223819.0,3222927.0,8.519394,9.617272,10.750402,701207.0,2.241385,True,True,True,3,Core Growth Market,Lower
1,UK,United Kingdom,individual_country,5274041.0,2373132.0,6.219928,7.021478,10.880105,517515.0,2.222397,True,True,True,3,Core Growth Market,Lower
2,NL,Netherlands,individual_country,11867612.0,5250119.0,13.996041,15.799683,3.659706,418986.0,2.260446,True,False,True,2,Core Retention Market,Lower
3,AT,Austria,individual_country,4360393.0,2131281.0,5.142419,5.805113,4.927288,204760.0,2.045902,True,True,False,2,Stay Extension Opportunity,High
4,PL,Poland,individual_country,4167089.0,1471375.0,4.914447,5.547762,0.155626,6475.0,2.832105,True,False,True,2,Core Retention Market,Lower



Source Roles
['strategic_role', 'markets', 'foreign_nights', 'germany_foreign_nights_share_pct', 'absolute_growth_nights', 'median_growth_pct', 'median_los']


,strategic_role,markets,foreign_nights,germany_foreign_nights_share_pct,absolute_growth_nights,median_growth_pct,median_los
0,Core Retention Market,2,16034701.0,18.910488,425461.0,1.907666,2.546276
1,Monitor / Maintain,4,14477008.0,17.073426,367721.0,2.756137,2.036068
2,Core Growth Market,2,12497860.0,14.739322,1218722.0,10.815253,2.231891
3,Stay Extension Opportunity,3,11538397.0,13.607782,528556.0,4.836640,2.045902
4,Emerging Growth Market,2,3946395.0,4.654172,379928.0,12.799503,2.466476



Travel BOP
['year', 'travel_balance_mio_eur', 'travel_receipts_mio_eur', 'travel_expenditure_mio_eur', 'receipts_expenditure_ratio', 'receipts_cover_expenditure_pct', 'is_provisional', 'receipts_growth_pct', 'expenditure_growth_pct', 'balance_change_mio_eur', 'receipts_change_mio_eur', 'expenditure_change_mio_eur']


,year,travel_balance_mio_eur,travel_receipts_mio_eur,travel_expenditure_mio_eur,receipts_expenditure_ratio,receipts_cover_expenditure_pct,is_provisional,receipts_growth_pct,expenditure_growth_pct,balance_change_mio_eur,receipts_change_mio_eur,expenditure_change_mio_eur
0,2021,-24300.0,18827.0,43126.0,0.436558,43.655799,False,NaN,NaN,NaN,NaN,NaN
1,2022,-54762.0,30257.0,85019.0,0.355885,35.588516,False,60.710681,97.140936,-30462.0,11430.0,41893.0
2,2023,-71650.0,34992.0,106642.0,0.328126,32.812588,False,15.649271,25.433138,-16888.0,4735.0,21623.0
3,2024,-69767.0,37055.0,106822.0,0.346885,34.688547,False,5.895633,0.168789,1883.0,2063.0,180.0
4,2025,-76822.0,37772.0,114594.0,0.329616,32.961586,True,1.934962,7.275655,-7055.0,717.0,7772.0



2026 YTD
['year', 'foreign_arrivals', 'foreign_nights', 'avg_length_of_stay', 'arrivals_growth_pct', 'nights_growth_pct', 'los_change_pct', 'arrivals_change', 'nights_change', 'los_change_nights']


,year,foreign_arrivals,foreign_nights,avg_length_of_stay,arrivals_growth_pct,nights_growth_pct,los_change_pct,arrivals_change,nights_change,los_change_nights
0,2025,16020024.0,35984384.0,2.246213,NaN,NaN,NaN,NaN,NaN,NaN
1,2026,16346103.0,36249231.0,2.217607,2.035446,0.736005,-1.273519,326079.0,264847.0,-0.028606



ML Scenario
['indicator', 'value']


,indicator,value
0,H1 2026 Actual,3.624923e+07
1,H2 2026 Seasonal Naive Projection,4.709106e+07
2,Full-Year 2026 Baseline Scenario,8.334029e+07
3,Full-Year 2025 Actual,8.307544e+07
4,Implied 2026 Growth (%),3.188030e-01


In [4]:
germany_position = (
    eu_benchmark[
        eu_benchmark[
            "country_code"
        ] == "DE"
    ]
    .copy()
)

display(
    germany_position
)

,country_code,country,foreign_arrivals,arrivals_rank,foreign_nights,nights_rank,avg_length_of_stay,los_rank,scale_position,stay_position,competitive_position
5,DE,Germany,37419799.0,4,84792636.0,7,2.265983,21,High Scale,Shorter Stay,High Scale / Shorter Stay


In [5]:
germany_2024 = (
    germany_position
    .iloc[0]
)

ytd_2026 = (
    ytd_summary[
        ytd_summary[
            "year"
        ] == 2026
    ]
    .iloc[0]
)

baseline_2026_growth = (
    ml_scenario.loc[
        ml_scenario[
            "indicator"
        ] == "Implied 2026 Growth (%)",
        "value"
    ]
    .iloc[0]
)

strategic_evidence = pd.DataFrame({
    "dimension": [
        "Visitor Scale",
        "Overnight Scale",
        "Stay Duration",
        "2026 Visitor Momentum",
        "2026 Overnight Momentum",
        "2026 Stay Duration",
        "2026 Baseline Scenario"
    ],

    "indicator": [
        "2024 EU arrivals rank",
        "2024 EU nights rank",
        "2024 EU LOS rank",
        "H1 arrivals growth",
        "H1 nights growth",
        "H1 LOS change",
        "Implied full-year nights growth"
    ],

    "value": [
        germany_2024[
            "arrivals_rank"
        ],
        germany_2024[
            "nights_rank"
        ],
        germany_2024[
            "los_rank"
        ],
        ytd_2026[
            "arrivals_growth_pct"
        ],
        ytd_2026[
            "nights_growth_pct"
        ],
        ytd_2026[
            "los_change_pct"
        ],
        baseline_2026_growth
    ],

    "strategic_signal": [
        "Strength",
        "Strength",
        "Constraint",
        "Positive",
        "Modest",
        "Negative",
        "Modest"
    ]
})

display(
    strategic_evidence
)

,dimension,indicator,value,strategic_signal
0,Visitor Scale,2024 EU arrivals rank,4.000000,Strength
1,Overnight Scale,2024 EU nights rank,7.000000,Strength
2,Stay Duration,2024 EU LOS rank,21.000000,Constraint
3,2026 Visitor Momentum,H1 arrivals growth,2.035446,Positive
4,2026 Overnight Momentum,H1 nights growth,0.736005,Modest
5,2026 Stay Duration,H1 LOS change,-1.273519,Negative
6,2026 Baseline Scenario,Implied full-year nights growth,0.318803,Modest


## Germany's Core Competitive Position

Germany demonstrates a clear distinction between **tourism scale** and **tourism depth**.

In the complete 2024 EU27 benchmark, Germany ranked fourth in international accommodation arrivals and seventh in international overnight stays, confirming its position as one of Europe's major international tourism destinations.

However, Germany's average length of stay was approximately 2.27 nights, ranking only 21st among the 27 EU countries. Germany is therefore best characterized as a **high-scale, shorter-stay destination**.

This distinction is strategically important. Germany's principal competitive challenge is not simply attracting international visitors. It is converting its already substantial visitor volume into greater overnight-stay depth.

The 2026 year-to-date evidence reinforces this interpretation. During January–June 2026, international arrivals increased by approximately 2.04% compared with the same period in 2025, while overnight stays increased by only 0.74%. Average length of stay declined by approximately 1.27%.

The predictive analysis points to similarly modest near-term overnight-stay momentum. Under the validated seasonal-naive baseline scenario, full-year 2026 international overnight stays would reach approximately 83.34 million, only around 0.32% above the 2025 level.

Taken together, the evidence suggests that Germany remains highly competitive in attracting international visitor volume, but its stronger strategic opportunity lies in improving the duration and depth of those visits rather than relying exclusively on additional arrivals.

In [7]:
stay_opportunity = (
    los_counterfactual[
        [
            "benchmark",
            "benchmark_los",
            "illustrative_nights",
            "additional_nights_vs_actual",
            "additional_nights_pct"
        ]
    ]
    .copy()
)

display(
    stay_opportunity.round(2)
)

,benchmark,benchmark_los,illustrative_nights,additional_nights_vs_actual,additional_nights_pct
0,Germany Actual,2.27,8.479264e+07,0.00,0.00
1,EU27 Median,2.49,9.317841e+07,8385769.85,9.89
2,France,2.58,9.643208e+07,11639439.46,13.73
3,Netherlands,2.89,1.083015e+08,23508872.06,27.73
4,Austria,3.30,1.234376e+08,38644937.75,45.58
5,Weighted EU27,3.34,1.249956e+08,40202976.00,47.41


In [8]:
stay_opportunity_strategy = (
    stay_opportunity[
        stay_opportunity[
            "benchmark"
        ] != "Germany Actual"
    ]
    .copy()
)

stay_opportunity_strategy[
    "strategic_use"
] = (
    stay_opportunity_strategy[
        "benchmark"
    ]
    .map({
        "EU27 Median":
            "Conservative reference",
        "France":
            "Comparable major-destination reference",
        "Netherlands":
            "Higher-stay reference",
        "Austria":
            "Regional high-depth reference",
        "Weighted EU27":
            "EU-wide volume-weighted reference"
    })
)

display(
    stay_opportunity_strategy
)

,benchmark,benchmark_los,illustrative_nights,additional_nights_vs_actual,additional_nights_pct,strategic_use
1,EU27 Median,2.490083,9.317841e+07,8.385770e+06,9.889738,Conservative reference
2,France,2.577033,9.643208e+07,1.163944e+07,13.726946,Comparable major-destination reference
3,Netherlands,2.894230,1.083015e+08,2.350887e+07,27.725134,Higher-stay reference
4,Austria,3.298724,1.234376e+08,3.864494e+07,45.575818,Regional high-depth reference
5,Weighted EU27,3.340360,1.249956e+08,4.020298e+07,47.413287,EU-wide volume-weighted reference


## Tourism Depth Opportunity

Germany's comparatively short average stay creates a potentially important strategic lever.

The benchmark analysis shows that Germany does not need to approach the longest-stay destinations in Europe for stay duration to become economically meaningful. Even the more conservative scenario of matching the EU27 median average length of stay would correspond, at Germany's 2024 international arrival volume, to approximately 8.39 million additional overnight stays, or about 9.9% above the actual 2024 level.

More ambitious benchmarks produce substantially larger illustrative differences. These scenarios demonstrate the sensitivity of total overnight stays to average stay duration in a high-volume destination such as Germany.

However, these calculations are counterfactual benchmarks rather than forecasts. They do not establish that the additional nights are directly recoverable or that increasing average length of stay would occur automatically through a particular intervention.

Their strategic value is instead to demonstrate that, because Germany already attracts a large international visitor base, relatively modest improvements in stay duration can have meaningful implications for total tourism activity.

In [9]:
role_action_map = {
    "Core Growth Market":
        "Protect growth while strengthening repeat visitation and trip extension",

    "Core Retention Market":
        "Defend market share and maintain strong visitor volume",

    "Stay Extension Opportunity":
        "Prioritize longer-stay products, multi-destination itineraries, and trip extension",

    "Emerging Growth Market":
        "Support growth selectively and monitor whether momentum is sustained",

    "Growth Market - Shorter Stay":
        "Capture growth while actively testing stay-extension opportunities",

    "Long-Stay Niche":
        "Protect high-stay-value segments and develop targeted niche propositions",

    "Monitor / Maintain":
        "Maintain presence and monitor for changes in scale, growth, or stay duration"
}

source_market_actions = (
    source_markets[
        [
            "source_market_code",
            "source_market",
            "foreign_nights",
            "germany_foreign_nights_share_pct",
            "growth_2023_2024_pct",
            "absolute_growth_nights",
            "avg_length_of_stay",
            "strategic_role",
            "stay_extension_priority"
        ]
    ]
    .copy()
)

source_market_actions[
    "recommended_action"
] = (
    source_market_actions[
        "strategic_role"
    ]
    .map(
        role_action_map
    )
)

display(
    source_market_actions
    .sort_values(
        "foreign_nights",
        ascending=False
    )
    .reset_index(
        drop=True
    )
)

,source_market_code,source_market,foreign_nights,germany_foreign_nights_share_pct,growth_2023_2024_pct,absolute_growth_nights,avg_length_of_stay,strategic_role,stay_extension_priority,recommended_action
0,NL,Netherlands,11867612.0,13.996041,3.659706,418986.0,2.260446,Core Retention Market,Lower,Defend market share and maintain strong visito...
1,US,United States,7223819.0,8.519394,10.750402,701207.0,2.241385,Core Growth Market,Lower,Protect growth while strengthening repeat visi...
2,CH_LI,Switzerland and Liechtenstein,6897129.0,8.134113,2.414773,162623.0,2.080067,Monitor / Maintain,High,Maintain presence and monitor for changes in s...
3,UK,United Kingdom,5274041.0,6.219928,10.880105,517515.0,2.222397,Core Growth Market,Lower,Protect growth while strengthening repeat visi...
4,AT,Austria,4360393.0,5.142419,4.927288,204760.0,2.045902,Stay Extension Opportunity,High,"Prioritize longer-stay products, multi-destina..."
5,PL,Poland,4167089.0,4.914447,0.155626,6475.0,2.832105,Core Retention Market,Lower,Defend market share and maintain strong visito...
6,FR,France,3680328.0,4.340386,4.836640,169792.0,1.990412,Stay Extension Opportunity,High,"Prioritize longer-stay products, multi-destina..."
7,IT,Italy,3497676.0,4.124976,4.605835,154004.0,2.217334,Stay Extension Opportunity,High,"Prioritize longer-stay products, multi-destina..."
8,DK,Denmark,3051481.0,3.598757,1.866048,55899.0,1.994744,Monitor / Maintain,Lower,Maintain presence and monitor for changes in s...
9,BE,Belgium,3044527.0,3.590556,3.558510,104617.0,2.077393,Monitor / Maintain,Lower,Maintain presence and monitor for changes in s...


In [10]:
strategic_role_summary = (
    source_market_actions
    .groupby(
        "strategic_role",
        as_index=False
    )
    .agg(
        markets=(
            "source_market_code",
            "count"
        ),

        foreign_nights=(
            "foreign_nights",
            "sum"
        ),

        germany_nights_share_pct=(
            "germany_foreign_nights_share_pct",
            "sum"
        ),

        absolute_growth_nights=(
            "absolute_growth_nights",
            "sum"
        ),

        median_growth_pct=(
            "growth_2023_2024_pct",
            "median"
        ),

        median_los=(
            "avg_length_of_stay",
            "median"
        )
    )
)

strategic_role_summary = (
    strategic_role_summary
    .sort_values(
        "foreign_nights",
        ascending=False
    )
    .reset_index(
        drop=True
    )
)

display(
    strategic_role_summary.round(2)
)

,strategic_role,markets,foreign_nights,germany_nights_share_pct,absolute_growth_nights,median_growth_pct,median_los
0,Core Retention Market,2,16034701.0,18.91,425461.0,1.91,2.55
1,Monitor / Maintain,4,14477008.0,17.07,367721.0,2.76,2.04
2,Core Growth Market,2,12497860.0,14.74,1218722.0,10.82,2.23
3,Stay Extension Opportunity,3,11538397.0,13.61,528556.0,4.84,2.05
4,Emerging Growth Market,2,3946395.0,4.65,379928.0,12.80,2.47
5,Long-Stay Niche,2,2844353.0,3.35,70367.0,2.53,2.84
6,Growth Market - Shorter Stay,1,1824786.0,2.15,526494.0,40.55,2.17


In [11]:
stay_extension_markets = (
    source_market_actions[
        source_market_actions[
            "stay_extension_priority"
        ].isin(
            [
                "High",
                "Emerging"
            ]
        )
    ]
    .copy()
)

stay_extension_markets = (
    stay_extension_markets
    .sort_values(
        [
            "stay_extension_priority",
            "foreign_nights"
        ],
        ascending=[
            True,
            False
        ]
    )
    .reset_index(
        drop=True
    )
)

display(
    stay_extension_markets
)

,source_market_code,source_market,foreign_nights,germany_foreign_nights_share_pct,growth_2023_2024_pct,absolute_growth_nights,avg_length_of_stay,strategic_role,stay_extension_priority,recommended_action
0,CN,China,1824786.0,2.152057,40.552819,526494.0,2.167343,Growth Market - Shorter Stay,Emerging,Capture growth while actively testing stay-ext...
1,CH_LI,Switzerland and Liechtenstein,6897129.0,8.134113,2.414773,162623.0,2.080067,Monitor / Maintain,High,Maintain presence and monitor for changes in s...
2,AT,Austria,4360393.0,5.142419,4.927288,204760.0,2.045902,Stay Extension Opportunity,High,"Prioritize longer-stay products, multi-destina..."
3,FR,France,3680328.0,4.340386,4.836640,169792.0,1.990412,Stay Extension Opportunity,High,"Prioritize longer-stay products, multi-destina..."
4,IT,Italy,3497676.0,4.124976,4.605835,154004.0,2.217334,Stay Extension Opportunity,High,"Prioritize longer-stay products, multi-destina..."


## Source-Market Strategic Priorities

Germany's international source markets should not be managed through a single acquisition strategy. The analysis identifies distinct market roles based on current scale, recent growth, and average length of stay.

### Core Growth Markets

The United States and United Kingdom combine substantial overnight-stay volume with strong recent growth. These markets should remain important acquisition and retention priorities. However, because Germany's overall strategic challenge concerns tourism depth, growth in these markets should also be accompanied by efforts to encourage broader itineraries and repeat visitation rather than focusing exclusively on additional arrivals.

### Core Retention Markets

The Netherlands and Poland represent large established sources of overnight stays. Their strategic value lies primarily in protecting existing volume and maintaining Germany's competitive position. Poland is additionally notable for a comparatively longer average stay among Germany's major source markets.

### Stay Extension Opportunities

Austria, France, and Italy combine meaningful current scale with comparatively short average stays. These markets are therefore particularly relevant to Germany's tourism-depth challenge.

Rather than treating additional visitor acquisition as the only objective, destination marketing and tourism-product development for these markets could emphasize multi-city travel, regional extensions, cultural routes, events, rail-connected itineraries, and other propositions designed to increase the number of nights associated with an existing trip.

The Switzerland and Liechtenstein combined market also has high stay-extension relevance because of its substantial contribution to Germany's international overnight stays and comparatively short average stay.

### Emerging Growth Opportunities

China represents a different opportunity profile. It recorded particularly strong recent growth but remains a shorter-stay market. The strategic objective should therefore be two-dimensional: determine whether the renewed growth is sustained while simultaneously testing whether itineraries can be extended beyond short visits.

### Long-Stay Niches

Markets already associated with relatively long stays should not automatically be treated as primary acquisition targets. Instead, they provide evidence that some visitor segments already support greater tourism depth. These markets may therefore be useful for targeted retention, niche product development, and understanding the characteristics associated with longer stays.

Overall, the evidence supports a differentiated source-market strategy: protect established high-volume markets, selectively accelerate markets with demonstrated growth, and use stay-extension initiatives where large visitor bases are not translating into equally strong overnight-stay depth.

In [12]:
economic_context = (
    travel_bop[
        [
            "year",
            "travel_receipts_mio_eur",
            "travel_expenditure_mio_eur",
            "travel_balance_mio_eur",
            "receipts_cover_expenditure_pct",
            "receipts_growth_pct",
            "expenditure_growth_pct",
            "is_provisional"
        ]
    ]
    .copy()
)

display(
    economic_context.round(2)
)

,year,travel_receipts_mio_eur,travel_expenditure_mio_eur,travel_balance_mio_eur,receipts_cover_expenditure_pct,receipts_growth_pct,expenditure_growth_pct,is_provisional
0,2021,18827.0,43126.0,-24300.0,43.66,NaN,NaN,False
1,2022,30257.0,85019.0,-54762.0,35.59,60.71,97.14,False
2,2023,34992.0,106642.0,-71650.0,32.81,15.65,25.43,False
3,2024,37055.0,106822.0,-69767.0,34.69,5.90,0.17,False
4,2025,37772.0,114594.0,-76822.0,32.96,1.93,7.28,True


In [13]:
bop_2021 = (
    economic_context[
        economic_context["year"] == 2021
    ]
    .iloc[0]
)

bop_2025 = (
    economic_context[
        economic_context["year"] == 2025
    ]
    .iloc[0]
)

receipts_change_2021_2025_pct = (
    (
        bop_2025["travel_receipts_mio_eur"]
        / bop_2021["travel_receipts_mio_eur"]
    )
    - 1
) * 100

expenditure_change_2021_2025_pct = (
    (
        bop_2025["travel_expenditure_mio_eur"]
        / bop_2021["travel_expenditure_mio_eur"]
    )
    - 1
) * 100

balance_change_2021_2025 = (
    bop_2025["travel_balance_mio_eur"]
    - bop_2021["travel_balance_mio_eur"]
)

print(
    f"Receipts change 2021-2025: "
    f"{receipts_change_2021_2025_pct:.2f}%"
)

print(
    f"Expenditure change 2021-2025: "
    f"{expenditure_change_2021_2025_pct:.2f}%"
)

print(
    f"Travel balance change 2021-2025: "
    f"{balance_change_2021_2025:,.0f} MIO EUR"
)

Receipts change 2021-2025: 100.63%
Expenditure change 2021-2025: 165.72%
Travel balance change 2021-2025: -52,522 MIO EUR


## Travel-Economic Context

Germany's inbound travel receipts recovered strongly between 2021 and 2025, but outbound travel expenditure increased considerably faster.

Travel receipts increased from approximately €18.8 billion in 2021 to €37.8 billion in 2025, while travel expenditure increased from approximately €43.1 billion to €114.6 billion. As a result, Germany's travel balance became substantially more negative over the period.

The 2025 figures should be interpreted cautiously because they are provisional.

The negative travel balance does **not** mean that Germany's tourism industry is unprofitable. The balance-of-payments travel account compares money received from non-residents travelling in Germany with money spent abroad by German residents. It therefore provides a national external travel-economy indicator rather than a measure of accommodation-sector profitability.

For tourism strategy, the relevant finding is that stronger inbound tourism receipts remain economically valuable even in a country with substantial outbound travel expenditure. Germany's high international visitor volume therefore creates an opportunity to deepen inbound tourism activity through longer stays and broader visitor consumption.

These economic indicators are not directly combined with accommodation arrivals or overnight stays because the datasets represent different statistical systems and scopes.

In [14]:
integrated_diagnosis = pd.DataFrame({
    "strategic_dimension": [
        "International Visitor Scale",
        "International Overnight Scale",
        "Average Length of Stay",
        "Source-Market Growth",
        "Stay-Extension Potential",
        "2026 Current Momentum",
        "Travel-Economic Context",
        "Predictive Outlook"
    ],

    "evidence": [
        "4th in EU27 arrivals in 2024",
        "7th in EU27 foreign nights in 2024",
        "2.27 nights and 21st in EU27 in 2024",
        "US and UK strong established growth; China rapid emerging growth",
        "CH_LI, Austria, France, Italy and China identified for stay-extension attention",
        "H1 arrivals +2.04%, nights +0.74%, LOS -1.27%",
        "Inbound receipts recovered, but outbound expenditure grew faster",
        "Seasonal-naive baseline implies approximately +0.32% full-year nights growth"
    ],

    "assessment": [
        "Competitive Strength",
        "Competitive Strength",
        "Primary Constraint",
        "Selective Opportunity",
        "Strategic Opportunity",
        "Modest / Mixed",
        "Economic Opportunity with External Imbalance",
        "Modest Baseline"
    ],

    "strategic_implication": [
        "Protect Germany's strong international visitor position",
        "Maintain overnight-stay scale while improving conversion from arrivals to nights",
        "Prioritize tourism depth and trip extension",
        "Use differentiated acquisition rather than uniform market expansion",
        "Target suitable source markets with longer-stay propositions",
        "Do not rely on visitor growth alone to increase tourism depth",
        "Strengthening inbound tourism activity remains economically relevant",
        "Plan around modest baseline growth rather than assuming rapid expansion"
    ]
})

display(
    integrated_diagnosis
)

,strategic_dimension,evidence,assessment,strategic_implication
0,International Visitor Scale,4th in EU27 arrivals in 2024,Competitive Strength,Protect Germany's strong international visitor...
1,International Overnight Scale,7th in EU27 foreign nights in 2024,Competitive Strength,Maintain overnight-stay scale while improving ...
2,Average Length of Stay,2.27 nights and 21st in EU27 in 2024,Primary Constraint,Prioritize tourism depth and trip extension
3,Source-Market Growth,US and UK strong established growth; China rap...,Selective Opportunity,Use differentiated acquisition rather than uni...
4,Stay-Extension Potential,"CH_LI, Austria, France, Italy and China identi...",Strategic Opportunity,Target suitable source markets with longer-sta...
5,2026 Current Momentum,"H1 arrivals +2.04%, nights +0.74%, LOS -1.27%",Modest / Mixed,Do not rely on visitor growth alone to increas...
6,Travel-Economic Context,"Inbound receipts recovered, but outbound expen...",Economic Opportunity with External Imbalance,Strengthening inbound tourism activity remains...
7,Predictive Outlook,Seasonal-naive baseline implies approximately ...,Modest Baseline,Plan around modest baseline growth rather than...


# Integrated Strategic Diagnosis

The evidence presents Germany as a **large and resilient international tourism destination with a tourism-depth constraint rather than a visitor-scale constraint**.

Germany's fourth-place EU ranking in international arrivals demonstrates strong visitor acquisition, while its seventh-place position in overnight stays confirms substantial tourism volume. However, its 21st-place average length-of-stay ranking shows that this visitor scale does not translate proportionally into overnight-stay depth.

The source-market analysis demonstrates that this challenge should not be addressed through a uniform international marketing strategy. The United States and United Kingdom offer established growth opportunities, the Netherlands and Poland require retention, Austria, France and Italy present stronger stay-extension relevance, and China combines rapid recent growth with shorter-stay characteristics. The Switzerland and Liechtenstein combined market also warrants stay-extension attention because of its substantial existing volume and relatively short average stay.

Current performance reinforces the same diagnosis. International arrivals continued to increase during the first half of 2026, but overnight stays grew more slowly and average length of stay declined further. The seasonal-naive baseline scenario also suggests only modest full-year overnight-stay growth if the second half of 2026 broadly repeats the 2025 seasonal pattern.

The economic evidence adds a broader perspective. Inbound travel receipts have recovered substantially, but Germany continues to record much larger outbound travel expenditure. This does not indicate tourism-sector unprofitability, but it reinforces the economic relevance of strengthening inbound tourism activity.

Germany's strategic question is therefore not simply:

**How can Germany attract more international visitors?**

A more useful question is:

**How can Germany create greater tourism value and overnight-stay depth from the large international visitor base it already attracts, while selectively growing markets with demonstrated potential?**

In [15]:
strategic_recommendations = pd.DataFrame({
    "priority": [
        1,
        2,
        3,
        4
    ],

    "strategic_priority": [
        "Increase Stay Duration",
        "Differentiate Source-Market Strategy",
        "Convert Growth into Tourism Depth",
        "Monitor Performance with Leading Indicators"
    ],

    "evidence_basis": [
        "Germany ranks 21st in EU27 LOS despite ranking 4th in arrivals",
        "Major source markets differ substantially in scale, growth and LOS",
        "H1 2026 arrivals grew faster than nights while LOS declined",
        "Seasonal persistence is strong and the 2026 baseline indicates modest growth"
    ],

    "recommended_direction": [
        "Develop trip-extension, multi-city and regional itinerary propositions",
        "Separate growth, retention, stay-extension and niche market objectives",
        "Evaluate success using nights and LOS alongside visitor arrivals",
        "Track monthly arrivals, nights, LOS and source-market performance against seasonal benchmarks"
    ],

    "primary_business_kpi": [
        "Average Length of Stay",
        "Foreign Nights by Source Market",
        "Nights Growth Relative to Arrivals Growth",
        "Monthly Forecast Error and YTD Performance"
    ]
})

display(
    strategic_recommendations
)

,priority,strategic_priority,evidence_basis,recommended_direction,primary_business_kpi
0,1,Increase Stay Duration,Germany ranks 21st in EU27 LOS despite ranking...,"Develop trip-extension, multi-city and regiona...",Average Length of Stay
1,2,Differentiate Source-Market Strategy,Major source markets differ substantially in s...,"Separate growth, retention, stay-extension and...",Foreign Nights by Source Market
2,3,Convert Growth into Tourism Depth,H1 2026 arrivals grew faster than nights while...,Evaluate success using nights and LOS alongsid...,Nights Growth Relative to Arrivals Growth
3,4,Monitor Performance with Leading Indicators,Seasonal persistence is strong and the 2026 ba...,"Track monthly arrivals, nights, LOS and source...",Monthly Forecast Error and YTD Performance


In [16]:
implementation_framework = pd.DataFrame({
    "strategic_priority": [
        "Increase Stay Duration",
        "Differentiate Source-Market Strategy",
        "Convert Growth into Tourism Depth",
        "Monitor Performance with Leading Indicators"
    ],

    "target_area": [
        "High-volume shorter-stay visitor segments",
        "International source-market portfolio",
        "Markets generating additional arrivals",
        "Germany monthly international tourism performance"
    ],

    "example_actions": [
        "Multi-city itineraries, regional extensions, events, cultural routes and rail-linked packages",
        "Separate campaigns and objectives for growth, retention, stay-extension and niche markets",
        "Pair acquisition campaigns with trip-extension propositions and measure resulting overnight stays",
        "Maintain monthly dashboard comparing actual performance with prior-year and seasonal benchmarks"
    ],

    "primary_kpi": [
        "Average Length of Stay",
        "Foreign Nights by Source Market",
        "Nights Growth vs Arrivals Growth",
        "Monthly Foreign Nights"
    ],

    "supporting_kpi": [
        "Foreign Nights",
        "Source-Market Growth and LOS",
        "Average Length of Stay",
        "Forecast Error and YTD Growth"
    ],

    "desired_direction": [
        "Increase",
        "Role-specific improvement",
        "Nights growth approaches or exceeds arrivals growth",
        "Actual performance meets or exceeds seasonal baseline"
    ]
})

display(
    implementation_framework
)

,strategic_priority,target_area,example_actions,primary_kpi,supporting_kpi,desired_direction
0,Increase Stay Duration,High-volume shorter-stay visitor segments,"Multi-city itineraries, regional extensions, e...",Average Length of Stay,Foreign Nights,Increase
1,Differentiate Source-Market Strategy,International source-market portfolio,"Separate campaigns and objectives for growth, ...",Foreign Nights by Source Market,Source-Market Growth and LOS,Role-specific improvement
2,Convert Growth into Tourism Depth,Markets generating additional arrivals,Pair acquisition campaigns with trip-extension...,Nights Growth vs Arrivals Growth,Average Length of Stay,Nights growth approaches or exceeds arrivals g...
3,Monitor Performance with Leading Indicators,Germany monthly international tourism performance,Maintain monthly dashboard comparing actual pe...,Monthly Foreign Nights,Forecast Error and YTD Growth,Actual performance meets or exceeds seasonal b...


In [17]:
strategic_priority_matrix = pd.DataFrame({
    "priority": [
        1,
        2,
        3,
        4
    ],

    "strategic_priority": [
        "Increase Stay Duration",
        "Differentiate Source-Market Strategy",
        "Convert Growth into Tourism Depth",
        "Monitor Performance with Leading Indicators"
    ],

    "evidence_strength": [
        "High",
        "High",
        "High",
        "High"
    ],

    "strategic_urgency": [
        "High",
        "High",
        "High",
        "Medium"
    ],

    "reason": [
        "LOS is Germany's clearest competitive weakness and continued declining in H1 2026",
        "Source markets show materially different combinations of scale, growth and stay duration",
        "2026 arrivals are increasing faster than overnight stays",
        "Strong seasonality makes regular benchmark monitoring practical and informative"
    ]
})

display(
    strategic_priority_matrix
)

,priority,strategic_priority,evidence_strength,strategic_urgency,reason
0,1,Increase Stay Duration,High,High,LOS is Germany's clearest competitive weakness...
1,2,Differentiate Source-Market Strategy,High,High,Source markets show materially different combi...
2,3,Convert Growth into Tourism Depth,High,High,2026 arrivals are increasing faster than overn...
3,4,Monitor Performance with Leading Indicators,High,Medium,Strong seasonality makes regular benchmark mon...


# Strategic Limitations and Decision Boundaries

The strategic recommendations should be interpreted within the limitations of the available evidence.

### 1. Association does not establish causation

The statistical and source-market analyses identify relationships and patterns but do not demonstrate that a specific tourism intervention will cause visitors to stay longer or generate additional overnight stays.

### 2. Length-of-stay scenarios are counterfactual benchmarks

The additional-night scenarios show what Germany's overnight-stay volume would look like mathematically at alternative average lengths of stay while holding arrivals constant. They are not forecasts of recoverable demand and should not be interpreted as guaranteed tourism gains.

### 3. Source-market classifications are decision-support categories

The growth, retention, stay-extension and niche classifications are based on observed scale, growth and stay-duration characteristics. They provide a structured basis for prioritization but do not replace market-specific research into traveler motivation, accessibility, spending behavior or campaign feasibility.

### 4. 2026 evidence is incomplete

Observed 2026 tourism performance covers January through June only. It should therefore be interpreted as year-to-date evidence rather than a full-year result.

The July–December 2026 values used in the predictive analysis are seasonal-naive baseline projections, not observed Eurostat data.

### 5. Predictive evidence is based on a relatively small time series

The monthly modeling dataset covers the post-pandemic recovery period and contains limited observations. Although rolling-origin validation supports the seasonal-naive benchmark, unexpected economic, geopolitical or behavioral changes could cause future tourism patterns to differ from previous seasonal patterns.

### 6. Travel-economic and accommodation statistics have different scopes

Balance-of-payments travel statistics and accommodation arrivals or overnight stays represent different statistical systems. The economic indicators are therefore used as contextual evidence and are not directly converted into accommodation revenue or visitor-spending estimates.

### 7. Historical EU comparisons contain availability constraints

Cross-country historical rankings are based only on countries with complete comparable observations for the relevant period. The complete 2024 EU27 benchmark therefore provides the strongest cross-sectional basis for Germany's competitive positioning.

These limitations do not invalidate the strategic findings. Instead, they define the boundaries within which the evidence should be used for decision-making.

In [18]:
strategic_evidence.to_csv(
    processed_dir
    / "germany_strategic_evidence_summary.csv",
    index=False
)

stay_opportunity_strategy.to_csv(
    processed_dir
    / "germany_stay_duration_opportunity_strategy.csv",
    index=False
)

source_market_actions.to_csv(
    processed_dir
    / "germany_source_market_action_matrix.csv",
    index=False
)

strategic_role_summary.to_csv(
    processed_dir
    / "germany_strategic_role_summary.csv",
    index=False
)

stay_extension_markets.to_csv(
    processed_dir
    / "germany_stay_extension_priority_markets.csv",
    index=False
)

economic_context.to_csv(
    processed_dir
    / "germany_strategic_economic_context.csv",
    index=False
)

integrated_diagnosis.to_csv(
    processed_dir
    / "germany_integrated_strategic_diagnosis.csv",
    index=False
)

strategic_recommendations.to_csv(
    processed_dir
    / "germany_strategic_recommendations.csv",
    index=False
)

implementation_framework.to_csv(
    processed_dir
    / "germany_strategy_implementation_framework.csv",
    index=False
)

strategic_priority_matrix.to_csv(
    processed_dir
    / "germany_strategic_priority_matrix.csv",
    index=False
)

print(
    "Notebook 10 processed outputs saved."
)

Notebook 10 processed outputs saved.


In [19]:
output_files = [
    "germany_strategic_evidence_summary.csv",
    "germany_stay_duration_opportunity_strategy.csv",
    "germany_source_market_action_matrix.csv",
    "germany_strategic_role_summary.csv",
    "germany_stay_extension_priority_markets.csv",
    "germany_strategic_economic_context.csv",
    "germany_integrated_strategic_diagnosis.csv",
    "germany_strategic_recommendations.csv",
    "germany_strategy_implementation_framework.csv",
    "germany_strategic_priority_matrix.csv"
]

for file_name in output_files:
    file_path = (
        processed_dir
        / file_name
    )

    print(
        file_name,
        "->",
        file_path.exists()
    )

germany_strategic_evidence_summary.csv -> True
germany_stay_duration_opportunity_strategy.csv -> True
germany_source_market_action_matrix.csv -> True
germany_strategic_role_summary.csv -> True
germany_stay_extension_priority_markets.csv -> True
germany_strategic_economic_context.csv -> True
germany_integrated_strategic_diagnosis.csv -> True
germany_strategic_recommendations.csv -> True
germany_strategy_implementation_framework.csv -> True
germany_strategic_priority_matrix.csv -> True


# Conclusion

The strategic analysis identifies Germany as a **high-scale international tourism destination whose principal competitive opportunity lies in increasing tourism depth**.

Germany's fourth-place EU27 ranking in international accommodation arrivals and seventh-place ranking in international overnight stays demonstrate substantial international tourism scale. However, its average length of stay of approximately 2.27 nights ranked only 21st in the complete 2024 EU27 comparison. This gap between visitor scale and stay duration is the central strategic finding of the project.

The source-market analysis shows that the response should be differentiated rather than uniform. The United States and United Kingdom represent core growth markets, while the Netherlands and Poland require protection of established volume. Austria, France and Italy present stronger stay-extension opportunities, while the Switzerland and Liechtenstein combined market is also relevant because of its substantial scale and comparatively short stays. China represents a distinct emerging opportunity because rapid recent growth is combined with shorter stay duration.

The 2026 evidence reinforces the tourism-depth challenge. During January–June 2026, international arrivals increased by approximately 2.04%, but overnight stays increased by only 0.74%, while average length of stay declined by approximately 1.27%. This indicates that additional visitors are not currently translating proportionally into additional overnight stays.

Predictive modeling also suggests that rapid near-term expansion should not be assumed. A seasonal-naive benchmark outperformed both Linear Regression and Random Forest in chronological and rolling-origin validation. Under the resulting conditional baseline scenario, Germany would record approximately 83.34 million international overnight stays in 2026, only around 0.32% above the 2025 level.

The strategic implication is therefore not to abandon visitor acquisition, but to broaden the definition of tourism growth. Germany should protect its strong visitor base, selectively develop source markets with demonstrated growth potential, and place greater emphasis on converting visitor volume into longer and deeper stays.

Four priorities follow from the evidence:

1. Increase average stay duration through trip-extension and multi-destination propositions.
2. Differentiate source-market strategies according to growth, retention, stay-extension and niche roles.
3. Evaluate tourism growth using overnight stays and average length of stay alongside arrivals.
4. Monitor monthly performance against seasonal benchmarks so that changes in visitor volume, stay duration and source-market behavior can be identified early.

Overall, Germany's competitive challenge is not primarily one of tourism visibility or international visitor scale. It is one of **converting an already strong international visitor position into greater overnight-stay depth and tourism value**.